### DISCUSSION TITLE
Treeviews

### DISCUSSION THREAD

**Discussion by rexkogitans** [libui-ng/libui-ng#128] [Ideas]
Is it just that I do not see it, or does LibUI not yet provide treeviews? Maybe I can help out with an implementation in GTK.

**Comment by cody271** [comment]
> Is it just that I do not see it, or does LibUI not yet provide treeviews?

That's correct: there is no tree control at all yet. See new issue: [here](https://github.com/libui-ng/libui-ng/issues/130).

> Maybe I can help out with an implementation in GTK.

PRs welcome!
